src/main/groovy/org/transmartproject/batch/concept/ConceptTree.groovy

```diff
@@ -54,10 +54,21 @@ class ConceptTree {
         }
     }
 
+    //TODO We should have just one way to check is record new @{see getConceptNodesToInsert}
```

Comment:
So why didn't you make it consistent? You added `getConceptNodesToInsert`, after all.